tango_simlib/sim_xmi_parser.py

```diff
@@ -596,6 +663,9 @@ def get_reformatted_properties_metadata(self, property_group):
 
         return properties
 
+    def get_reformatted_class_description_metadata(self):
+        return self.class_description[0]
```

Comment:
Using it this way, I wonder if it makes sense to have class_description as a list.  Is an XMI file ever going to have more than one class description?